acme/api/handler.go

```diff
@@ -59,13 +59,15 @@ func (h *Handler) Route(r api.Router) {
 
 	r.MethodFunc("POST", getLink(acme.NewAccountLink, "{provisionerID}", false, nil), extractPayloadByJWK(h.NewAccount))
 	r.MethodFunc("POST", getLink(acme.AccountLink, "{provisionerID}", false, nil, "{accID}"), extractPayloadByKid(h.GetUpdateAccount))
+	r.MethodFunc("POST", getLink(acme.KeyChangeLink, "{provisionerID}", false, nil, "{accID}"), extractPayloadByKid(h.NotImplemented))
```

Comment:
> Any reason to ExtractPayloadByKid if we're just going to respond 501? That method might return a 4xx instead of a 501. @dopey

Reply:
what?

Reply:
Oh, sorry, were you just copying my comment to the right place?

Reply:
I figure we do as much as we can until we hit the "not implemented" point. That way we don't expose anything that doesn't need to be.